GeosCore/hco_utilities_gc_mod.F90

```diff
@@ -1773,14 +1781,70 @@ SUBROUTINE Get_GC_Restart( Input_Opt, State_Chm, State_Grid, &
 
    ENDIF
 
+   !=========================================================================
+   ! Get delta pressure per grid box stored in restart file to allow mass
+   ! conservation across consecutive runs. Set DP_DRY_PREV to this value
+   ! since it is the delta pressure used to create the restart file rather
+   ! than the current meteorology.
+   !========================================================================
+   v_name_in_hemco = 'DELPDRY'
+   v_name_in_file  = 'Met_DELPDRY'
+
+   IF ( use_HEMCO_for_restart_read ) THEN
+      CALL HCO_GC_GetPtr( Input_Opt, State_Grid, TRIM( v_name_in_hemco ),  &
+           Ptr3D,     RC,         FOUND=FOUND_DELP_DRY )
+   ELSE
+      Temp4D_r8 = 0.0_fp
+      Ptr4D_r8 => Temp4D_r8
+      st4d   = (/  1,  1,  1,  1 /)
+      ct4d   = (/ State_Grid%NX, State_Grid%NY, State_Grid%NZ, 1 /)
+      CALL NcRd( Ptr4D_r8, fId, TRIM(v_name_in_file), st4d, ct4d )
+      FOUND = .TRUE.
```

Comment:
@lizzlei: Should `FOUND` be `FOUND_DELP_DRY` here?  Otherwise the next IF block won't execute even though we've read `Met_DELPDRY` directly from the restart file.